Samples~/Cinemachine Example Scenes/Scenes/2D/Zoom2D.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Cinemachine;
+using UnityEngine;
+
+public class Zoom2D : MonoBehaviour
```

Comment:
You have to add logic to counteract save during play.  You need to store the original value on start, and restore it on exit,.